ngraph_onnx/onnx_importer/utils/mapping.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
```

Comment:
Missing copyright notice.
Let's rename this file to `types.py`

Reply:
Fixed